packages/canary-web-components/src/components/ic-date-input/ic-date-input.tsx

```diff
@@ -173,6 +173,11 @@ export class DateInput {
    */
   @Prop() hideHelperText: boolean = false;
 
+  /**
+   * If `true`, the label and helper text will be visually hidden, but the required label will still be read out by screen readers.
```

Comment:
does this prop need to hide the helper text? there is already a hideHelperText prop

Reply:
feels intuitive to me that if you want a labelless field you'd want helper text hidden too - but as a design system i'd agree that we could offer more flexibility if this prop only hides the label.

Reply:
there is precedent for hideLabel also hiding the helpertext with ic-text-field. The issue is hideHelperText has already been implemented as another feature request on the date input, so this causes a bit of an inconsistency in the components

Reply:
Is this not a blocker then, if hideLabel is doing the same thing as in ic-text-field?  I'm happy to remove my Request Changes if you're okay with Date Picker having a partly redundant hideHelperText prop :)